scripts/globals/weaponskills/spirits_within.lua

```diff
@@ -42,13 +43,20 @@ function onUseWeaponSkill(player, target, wsID, TP, primary)
     end
 
     local damage = target:breathDmgTaken(WSC);
-    damage = damage * WEAPON_SKILL_POWER
+    if (damage > 0) then
+        if (attacker:getOffhandDmg() ~= 0 and attacker:getOffhandDmg() > 0) then
```

Comment:
I copied that from weaponskills.lua (minus the h2h skill check). I can change that though.